utils/sccli.py

```diff
@@ -0,0 +1,222 @@
+            system("systemctl stop kube-proxy")
+        if not system("systemctl is-active kubelet")[2]:
+            system("systemctl stop kubelet")
+        system("systemctl reset-failed kube-apiserver"
+               " kube-controller-manager kube-scheduler kube-proxy kubelet")
+        if not system("systemctl is-active etcd")[2]:
+            system("systemctl stop etcd")
+    elif service_name == "openshift":
+        system("systemctl disable openshift")
+        if not system("systemctl is-active openshift")[2]:
+            system("systemctl stop openshift")
+        system("systemctl reset-failed openshift")
+    elif service_name == "docker":
+        if not system("systemctl is-active docker")[2]:
+            system("systemctl stop docker")
+        system("systemctl reset-failed docker")
+
+def setup_kube_service_account_key():
+    system("mkdir -p /etc/pki/kube-apiserver/")
+    system("openssl genrsa -out /etc/pki/kube-apiserver/serviceaccount.key 2048")
+    system("sed -i.back '/KUBE_API_ARGS=*/c\KUBE_API_ARGS="
+           "\"--service_account_key_file=/etc/pki/kube-apiserver/serviceaccount.key\"'"
+           " /etc/kubernetes/apiserver")
+    system("sed -i.back '/KUBE_CONTROLLER_MANAGER_ARGS=*/c\KUBE_CONTROLLER_MANAGER_ARGS="
+           "\"--service_account_private_key_file=/etc/pki/kube-apiserver/serviceaccount.key\"'"
+           "/etc/kubernetes/controller-manager")
+
+# This is not used "need discussion"
+def vagrant_box_variant():
+    try:
+        with open("/etc/os-release") as fh:
+            for line in fh.readlines():
+                if "Container Development Kit" in line:
+                    return False
+        return True
+    except IOError as err:
+        return (err, 37)
+
+def pull_openshift_images():
+    docker_registry = os.getenv('DOCKER_REGISTRY', DOCKER_REGISTRY)
+    image_name = os.getenv('IMAGE_NAME', IMAGE_NAME)
+    image_tag = os.getenv('IMAGE_TAG', IMAGE_TAG)
+    if system(('sed -i.back "/^IMAGE=*/cIMAGE=\"%s/%s\:%s\""'
+            ' /etc/sysconfig/openshift_option') % (docker_registry, image_name, image_tag))[2]:
+        return ("Permisison denined: /etc/sysconfig/openshift_option", 37)
+    image_pull_list = ("{0}/{1}:{2} "
+                       "{0}/{1}-haproxy-router:{2} "
+                       "{0}/{1}-deployer:{2} "
+                       "{0}/{1}-docker-registry:{2} "
+                       "{0}/{1}-sti-builder:{2}").format(docker_registry, image_name, image_tag)
+    for image in image_pull_list.split():
+        if system("docker pull %s" % image)[2]:
+            return ("%s Not Pulled" % image, 111)
+    return ('', 0)
+
+def service_start(service_name):
+    if service_name == "kubernetes":
+        service_stop("openshift")
+        # This is required because openshift does create kubeconfig for 8443 port
+        subprocess.call("rm -fr /home/vagrant/.kube", shell=True)
+        subprocess.call("rm -fr /root/.kube", shell=True)
```

Comment:
@LalatenduMohanty If we have openshift running and want to switch to service of kubernetes then `kubectl get nodes` not work because openshift oc comand create a `.kube/config` file which have `:8443` port mapped and those should be deleted.